src/bidiMapper/domains/context/PreloadScriptStorage.ts

```diff
@@ -16,9 +16,9 @@
  */
 import {v4 as uuidv4} from 'uuid';
 
-import {CommonDataTypes, Script} from '../../../protocol/protocol';
+import {CommonDataTypes, Script} from '../../../protocol/protocol.js';
```

Comment:
Can it be done once throughout the whole project in one dedicated PR?

Reply:
Done: https://github.com/GoogleChromeLabs/chromium-bidi/commit/28e665442640cf1db9b114b993cc40b3a74aae06